### PR TITLE
Create tonje.md

### PR DESCRIPTION
**Pull request by tonjegul**

### CODE REVIEW

File: participants/tonje.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+Veldig relevant foredrag! :smiley:
```
Comment (danielvo) [inline participants/tonje.md:2]: *nodding*

**Review by danielvo — APPROVED** [review]
Excellent!

**Comment by emrobe** [comment]
Great stuff!

File: participants/tonje.md (hunk 1 above)
Comment (emrobe) [reply participants/tonje.md:2]: Yes please!

**Review by emrobe — APPROVED** [review]

**Pull request merged**